apps/test-low-memory.sh

```diff
@@ -11,16 +11,26 @@ if [ -n "$CIRCLECI" ]; then \
 fi && \
 
 node --max_old_space_size=4096 `npm bin`/grunt storybookTest && \
-if [-n "$CIRCLECI" ]; then \
+if [ -n "$CIRCLECI" ]; then \
     bash <(curl -s https://codecov.io/bash) -cF storybook; \
 fi && \
 
 node --max_old_space_size=4096 `npm bin`/grunt scratchTest && \
-if [-n "$CIRCLECI" ]; then \
+if [ -n "$CIRCLECI" ]; then \
     bash <(curl -s https://codecov.io/bash) -cF scratch; \
 fi && \
 
-node --max_old_space_size=4096 `npm bin`/grunt integrationTest && \
+LEVEL_TYPE='maze|turtle|bounce|calc|eval|flappy|studio' node --max_old_space_size=4096 `npm bin`/grunt integrationTest && \
```

Comment:
I have not. The startup time for each call to `grunt integrationTest` is about 25 seconds on my laptop. I've opened https://github.com/code-dot-org/code-dot-org/pull/17200 to see how this performs on circle, and if it looks good, to merge as a subsequent PR.